### ISSUE TITLE
Manual Scraping not saving.  Tests results include and should be a simple fix :)

### ISSUE THREAD

**Issue by nickdj** [digitalLumberjack/recalbox-os#234]
I've been running a few tests with interesting results.

For me, scraping with the menu option works fine.  I select the matching game and after exiting the scraper I can see the amendment in the gamelist.xml file. 

However, and this bit was quite interesting, if you scrape a game manually (via the select button) and you do the same it does not amend the gamelist.xml file or create one if it doesn't exist.  If you then go back to the scraper in the menu and scrape just one game, the gamelist.xml updates with that game and the details of the games you scraped manually!  (If you reboot Recalbox without doing this the manual scrapes are lost.)

In conclusion the gamelist.xml file is only saving after a 'menu scrape' so hopefully that's a simple fix.

**Comment by digitalLumberjack** [comment]
Should be fixed with https://github.com/digitalLumberjack/recalbox-buildroot/commit/ed88abadf50c489e603e35dc63bb191448f13369 https://github.com/digitalLumberjack/recalbox-emulationstation/commit/0933a266cc7b4cf326c72b105867ef4dfa9a884d but we need to try that on 3.3.0

**Comment by lordtaki** [comment]
With 3.3.0 beta 1 (like 3.2.11) the manual scraping of a game (via select / edit / scrape / save) is not written to disk. The timestamp of the gamelist.xml is unchanged.

The only secure way to make it written is to force close emulationstation with F4 and monitor the end of the process (10 minutes for a pi2 with +20000 roms).

**Comment by mrclschstr** [comment]
I'm experiencing this issue with the latest 4.0.0-beta2. Currently I'm scraping manually but it would be nice, if this could be fixed! +1
